File: 200_code_python/10.py
# Viết bằng PYTHON
# Tìm số lớn nhất trong ba số

# Nhập ba số từ bàn phím
a = float(input("Nhap so thu nhat a: "))
b = float(input("Nhap so thu hai b: "))
c = float(input("Nhap so thu ba c: "))

# Tìm số lớn nhất
"""if a >= b and a >= c:
    so_lon_nhat = a
elif b >= a and b>= c:
    so_lon_nhat = b
else :
    so_lon_nhat = c"""
so_lon_nhat = max(a, b, c)

# In kết quả
print("So lon nhat là: ", so_lon_nhat)

File: 200_code_cpp/10.cpp
// Viết bằng C++
// Tìm số lớn nhất trong ba số

#include <iostream>
using namespace std;

int main() {
    // Khai báo biến
    float a, b, c;

    // Nhập ba số từ bàn phím
    cout << "Nhap so thu nhat a: ";
    cin >> a;
    cout << "Nhap so thu hai b: ";
    cin >> b;
    cout << "Nhap so thu ba c: ";
    cin >> c;

    // Tìm số lớn nhất
    float so_lon_nhat;

    if (a >= b && a >= c) {
        so_lon_nhat = a;
    } else if (b >= a && b >= c)
    {
        so_lon_nhat = b;
    } else {
        so_lon_nhat = c;
    }
    
    // In kết quả
    cout << "So lon nhat la: " << so_lon_nhat << endl;

    return 0;
}